### PR TITLE
MindSphere goes Insights Hub

### PR DESCRIPTION
**Pull request by StefanKellerSiemens**
Please assign this pull request to Mark van Ments and please do not merge before we release our new app store modules.
Planned release - early October.

### CODE REVIEW

File: content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-development-considerations.md [hunk 1]
```diff
@@ -181,82 +181,82 @@ To ensure that the correct application credentials are requested, you have to se
...
+## 6 Insights Hub Icons {#atlasui}
 
-The **Siemens MindSphere Web Content** module includes two ways of including MindSphere icons in your app.
+The **Siemens Insights Hub Web Content** module includes two ways of including Insights Hub icons in your app.
 
-### 6.1 MindSphere Icons as SVGs
+### 6.1 Insights Hub Icons as 'glyphicons'
 
-You can select MindSphere icons from Siemens MindSphere Web Content to be displayed as SVGs in your application.
+You can select Insights Hub icons from Siemens Insights Hub Web Content to be displayed as SVGs in your application.
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-development-considerations.md]: You've changed the title to 'glyphicons' but still talk about SVGs in the text. Should these be consistent?

File: content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-module-details.md [hunk 2]
```diff
@@ -165,53 +165,53 @@ The JSON should contain the following information:
 * appCopyright – app owner’s name and year of publication
 * links – links to additional information about the app
 
-More information on the structure and content of this JSON object, together with sample JSON, can be found in [App Information](https://design.mindsphere.io/osbar/get-started.html#app-information), on the MindSphere developer site.
+More information on the structure and content of this JSON object, together with sample JSON, can be found in [App Information](https://design.mindsphere.io/osbar/get-started.html#app-information), on the Insights Hub developer site.
 
-## 4 Siemens MindSphere Web Content{#msthemepack}
+## 4 Siemens Insights Hub Web Content{#msthemepack}
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-module-details.md:170]: Above you rename this **Siemens Insights Hub Widgets**

File: content/en/docs/releasenotes/catalog/_index.md [hunk 3]
```diff
@@ -763,7 +763,7 @@ These release notes cover changes made to the [Catalog](/catalog/).
 
 #### Improvements
 
-* **Mindsphere** applications are now indicated with the display of the Mindsphere logo for all associated assets in the Catalog.
+* **Insights Hub** applications are now indicated with the display of the Insights Hub logo for all associated assets in the Catalog.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/catalog/_index.md]: Is this true? Has the logo in the Catalog been changed.

I think it is better to leave the release notes alone and add a new release note to say that MindSphere has been replaced by Insights Hub whenever the update is made to Catalog.

File: content/en/docs/appstore/modules/deep-link.md [hunk 4]
```diff
@@ -129,7 +129,7 @@ To view all the available deep link configurations and example URLs, add the **D
 
     * When the value is left empty, the default location is `login.html` (this file should be available in the theme folder).
     * When the login location ends with `=` (for example, in the case of Mendix SSO: `/openid/login?continuation=`), the original deep link location will be appended to the login location.
-    * When using the module with a MindSphere app, use `/mindspherelogin.html?redirect_uri=` as a login location (MindSphere SSO v2.0 and above is required).
+    * When using the module with a Siemens Insights Hub app, use `/sso-login.html?redirect_uri=` as a login location (Insights Hub SSO V2.0 and above is required, for versions below V4.0 use mindsphere-login.html).
```
Comment (MarkvanMents) [inline content/en/docs/appstore/modules/deep-link.md]: I find this confusing as we talk about Insights Hub SSO V2.0 and then talk about versions below V4.0.
Are we talking about two different modules here, or is there some other way we can make it clearer?

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Thanks for the PR @StefanKellerSiemens and @HeikoOsigus - there is a lot of work in there. Sorry I didn't do it earlier, but I was waiting for a response to my email question.

I've done a proofread which you can see in this commit: https://github.com/mendix/docs/pull/6484/commits/5c2ca9ee3f2ddacfa504d53a6191fd6b473e6eb9

I have also made a couple of comments below, which I'd like you to address.

Do we need a note somewhere about the renaming as existing MindSphere-enabled apps will still have the old names.
Is there a process they need to follow to upgrade to the new modules with the new names?

Please can you look at the changes and my questions and let me know whether any further changes are needed and when you would like this to be published.

File: content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-development-considerations.md (hunk 1 above)
Comment (StefanKellerSiemens) [reply content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-development-considerations.md]: Good catch, should be consistent, maybe we just remove the SVG part?

File: content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-module-details.md (hunk 2 above)
Comment (StefanKellerSiemens) [reply content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-module-details.md:170]: We have a theme which is called **Siemens Insights Hub Web Content** and we have widgets which has been called **Siemens MindSphere Web Widgets** and nowadays only **Siemens Insights Hub Widgets**. So we have two different things here, and the title fits for me.

File: content/en/docs/releasenotes/catalog/_index.md (hunk 3 above)
Comment (StefanKellerSiemens) [reply content/en/docs/releasenotes/catalog/_index.md]: OK, we can do as you suggesting.

File: content/en/docs/appstore/modules/deep-link.md (hunk 4 above)
Comment (StefanKellerSiemens) [reply content/en/docs/appstore/modules/deep-link.md]: We talk just about one module here. At least the version V2.0 is needed.
But if you use a version >= 2.0 < 4.0 (old ones) you have to use mindsphere-login.html and not sso-login.

Maybe we do not need to explain how it works with older version. What is the Mendix philosophy? Just talking about the latest and greatest?

Comment (MarkvanMents) [reply content/en/docs/appstore/modules/deep-link.md]: I think we should still talk about old versions, but probably rephrase it a bit to make it clearer (to me!)
```suggestion
    * When using the deep link module with a Siemens Insights Hub app, use `/sso-login.html?redirect_uri=` as the login location if you are using version 4.0 and above of Insights Hub SSO. For versions of Insights Hub SSO between version 2.0 and version 4.0 use `mindsphere-login.html` as the `redirect_uri`. Insights Hub SSO below version 2.0 does not support login location on deep links.
```

File: content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-development-considerations.md [hunk 5]
```diff
@@ -181,82 +181,82 @@ To ensure that the correct application credentials are requested, you have to se
...
```
Comment (MarkvanMents) [inline content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-development-considerations.md]:
```suggestion
You can select Insights Hub icons from Siemens Insights Hub Web Content to be displayed in your application.
```

File: content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-development-considerations.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-development-considerations.md]: 👍 see suggestion below.

File: content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-module-details.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/partners/siemens/mindsphere/mendix-on-mindsphere/mindsphere-module-details.md:170]: 👍 I'll mark this as resolved.

File: content/en/docs/releasenotes/catalog/_index.md (hunk 3 above)
Comment (MarkvanMents) [reply content/en/docs/releasenotes/catalog/_index.md]: Changes reverted in https://github.com/mendix/docs/pull/6484/commits/675e81f8f0c2ea90c53d0a82c03e161a11b0b038

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
@StefanKellerSiemens 
I've resolved most of the issues and made a couple of suggestions.
Can you review them and commit the changes if you are happy (or let me know and I will commit them).
I think we are ready to publish once those changes are committed.
Thanks for the PR and all the work you have put into it.
Mark

**Review by MarkvanMents — APPROVED** [review]
Ready to be published when artifacts are published.

**Comment by MarkvanMents** [comment]
Approval for publication given over email.

**Pull request merged**